tests/integration/states/test_pip_state.py

```diff
@@ -44,11 +44,16 @@
 import salt.utils.win_functions
 import salt.utils.win_runas
 from salt.modules.virtualenv_mod import KNOWN_BINARY_NAMES
+from salt.modules.cmdmod import run
 from salt.exceptions import CommandExecutionError
 
 # Import 3rd-party libs
 from salt.ext import six
 
+IS_WIN_SERVER = False
+if salt.utils.platform.is_windows():
+    IS_WIN_SERVER = 'serverstandard' in run('Get-WindowsEdition -Online', shell='powershell').lower()
+
```

Comment:
How about this:
```
import salt.modules.win_system as win_system
if salt.utils.platform.is_windows():
    IS_WIN_SERVER =  win_system.get_system_info()['os_type'] != 'Workstation'
```
Because shelling out to powershell is slow